truss-chains/truss_chains/framework.py

```diff
@@ -812,9 +817,23 @@ def _validate_remote_config(
         definitions.RemoteConfig,
     ):
         _collect_error(
-            f"{cls.entity_type}s must have a `{definitions.REMOTE_CONFIG_NAME}` class variable "
-            f"of type `{definitions.RemoteConfig}`. Got `{type(remote_config)}` "
-            f"for `{cls}`.",
+            f"{cls.entity_type}s must have a `{definitions.REMOTE_CONFIG_NAME}` "
+            f"class variable of type `{definitions.RemoteConfig}`. Got "
+            f"`{type(remote_config)}`.",
+            _ErrorKind.TYPE_ERROR,
+            location,
+        )
+
+
+def _validate_engine_builder_config(
```

Comment:
Should we validate any portions of the remote config given that the engine builder config is provided? Based on the yaml generation code, it seems like a lot of properties will be skipped / ignored if provided